lib/views/register_account_view.dart

```diff
@@ -0,0 +1,196 @@
+                  ),
+                ),
+              ),
+              SizedBox(height: deviceHeight * 0.012),
+              Visibility(
+                  visible: showPasswordHint,
+                  maintainAnimation: true,
+                  maintainSize: true,
+                  maintainState: true,
+                  child: Container(
+                      height: 24,
+                      width: 303,
+                      alignment: AlignmentDirectional.topStart,
+                      child: Text(
+                          '至少4個數字',
+                          style: TextStyle(
+                              color: 'F5F5DC'.toColor(),
+                              fontSize: 14,
+                              fontFamily: 'Serif'
+                          )
+                      )
+                  )
+              ),
+              Container(
+                  height: 60.0,
+                  width: 303.0,
+                  decoration: BoxDecoration(
+                      color: '7DDEF8'.toColor(),
+                      borderRadius: BorderRadius.circular(20),
+                      border: Border.all(width: 5.0, color: 'F5F5DC'.toColor())
+                  ),
+                  child: Padding(
+                    padding: EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 6.0),
+                    child: Focus(
+                      onFocusChange: (hasFocus){
+                        setState(() {
+                          showPasswordHint = !showPasswordHint;
+                        });
+                      },
+                      child: TextField(
+                        //maxLength: 6,
+                          decoration: InputDecoration(
+                            prefixIcon: Icon(
+                              Icons.lock,
+                              size: 30.0,
+                              color: '1C1B1F'.toColor(),
+                            ),
+                            hintText: '密碼',
+                            hintStyle: TextStyle(
+                                fontSize: 20.0,
+                                fontFamily: 'Serif',
+                                color: '013E6D'.toColor()
+                            ),
+                            enabledBorder: InputBorder.none,
+                          )
+                      ),
+                    ),
+                  )
+              ),
+              SizedBox(height: deviceHeight * 0.0343),
+              Container(
+                  height: 60.0,
+                  width: 303.0,
+                  decoration: BoxDecoration(
+                      color: '7DDEF8'.toColor(),
+                      borderRadius: BorderRadius.circular(20),
+                      border: Border.all(width: 5.0, color: 'F5F5DC'.toColor())
+                  ),
+                  child: Padding(
+                    padding: EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 6.0),
+                    child: TextField(
+                      //maxLength: 6,
```

Comment:
Dito. Delete unnecessary comment.

Reply:
deleted comments in this file and the logInView file